extensions/typescript-language-features/src/languageFeatures/jsxLinkedEditing.ts

```diff
@@ -0,0 +1,49 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { DocumentSelector } from '../configuration/documentSelector';
+import { API } from '../tsServer/api';
+import * as typeConverters from '../typeConverters';
+import { ClientCapability, ITypeScriptServiceClient } from '../typescriptService';
+import { conditionalRegistration, requireMinVersion, requireSomeCapability } from './util/dependentRegistration';
+
+class JsxLinkedEditingSupport implements vscode.LinkedEditingRangeProvider {
```

Comment:
Nit - you currently don't limit this to any JSX-specific file, so maybe we rename this to `LinkedEditingSupport` and just leave a comment about the main use-case being JSX tags?

Reply:
Thanks. Will follow up with these changes in a new PR